pkgs/by-name/mo/mozcdic-ut-alt-cannadic/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
```

Comment:
These packages all look extremely similar. Ever considered maybe writing a wrapper for them? (You can then expose them in merge-ut-dictionaries or somewhere else)

Reply:
Thanks for checking these out so quickly!
I think it is better to create a wrapper too, but It is very convenient having the auto-update script running. Is there a way to have the auto-update happen without creating multiple derivations?